### PR TITLE
TINY-3161: Added details about `toolbar_location` changes in 5.3.

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: TINY-3161

Description of Changes:
A new new `toolbar_location` mode was added called `auto`. This is now the default value. It doesn't impact iframe mode as it'll always have room to render above due to how it works. As such this only impacts inline mode.

This change will make it so that when the content is at the top of the document, the toolbar will switch to "bottom" mode instead so as to not block the content.

DOD:
- [x] nav.yml has been updated if applicable
- [x] file has been included where required if applicable
- [x] files removed have been deleted, not just excluded from the build if applicable

Review
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/configuration/toolbar-location.md [hunk 1]
```diff
@@ -4,11 +4,13 @@
 
 The `toolbar_location` option is used to position the [toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar) and [menubar]({{site.baseurl}}/configure/editor-appearance/#menubar). The default behavior is to position the toolbar and menu bar above the content area. Setting this option to `bottom` positions the toolbar and menu bar below the content area. This option works in [classic editor]({{site.baseurl}}/general-configuration-guide/use-tinymce-classic/) (Iframe) mode and inline mode.
 
+When using `auto`, the menu bar and toolbar will be positioned above the content area by default. However, if there will never be enough space above to render the components, the menu bar and toolbar will be positioned at the the bottom instead.
```
Comment (tylerkelly13) [inline _includes/configuration/toolbar-location.md:7]: This _might_ be a better way to phrase this?

>Setting this option to `auto` positions the toolbar and menu bar either:
>- Above the content area (default)
>- Below the content area if the editor does not have enough space within the current frame to render the components above the editor.

Comment (lnewson) [reply _includes/configuration/toolbar-location.md]: Fixed :+1:

File: _includes/configuration/toolbar-location.md [hunk 2]
```diff
@@ -4,11 +4,15 @@
 
 The `toolbar_location` option is used to position the [toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar) and [menubar]({{site.baseurl}}/configure/editor-appearance/#menubar). The default behavior is to position the toolbar and menu bar above the content area. Setting this option to `bottom` positions the toolbar and menu bar below the content area. This option works in [classic editor]({{site.baseurl}}/general-configuration-guide/use-tinymce-classic/) (Iframe) mode and inline mode.
 
+Setting this option to `auto` positions the toolbar and menu bar either:
+* Above the content area. (default)
+* Below the content area if the editor does not have enough space within the current document to render the components above the content area.
```
Comment (metricjs) [inline _includes/configuration/toolbar-location.md:9]: "Below the content area if there is not enough space above the editor to display the toolbar and menubar." ?

File: _includes/configuration/toolbar-location.md [hunk 3]
```diff
@@ -4,11 +4,15 @@
 
 The `toolbar_location` option is used to position the [toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar) and [menubar]({{site.baseurl}}/configure/editor-appearance/#menubar). The default behavior is to position the toolbar and menu bar above the content area. Setting this option to `bottom` positions the toolbar and menu bar below the content area. This option works in [classic editor]({{site.baseurl}}/general-configuration-guide/use-tinymce-classic/) (Iframe) mode and inline mode.
 
+Setting this option to `auto` positions the toolbar and menu bar either:
+* Above the content area. (default)
```
Comment (metricjs) [inline _includes/configuration/toolbar-location.md:8]: "Above the content area by default" or "Above the content area if there is space" or something?

File: _includes/configuration/toolbar-location.md (hunk 2 above)
Comment (lnewson) [reply _includes/configuration/toolbar-location.md:9]: I don't want to save above the editor, as the toolbar/menu bar is part of the editor which is confusing?

Comment (lnewson) [reply _includes/configuration/toolbar-location.md]: Fixed as discussed on slack

File: _includes/configuration/toolbar-location.md (hunk 3 above)
Comment (lnewson) [reply _includes/configuration/toolbar-location.md]: Fixed

**Pull request merged**